services/webhook/telegram.go

```diff
@@ -181,7 +182,7 @@ func (t telegramConvertor) Package(p *api.PackagePayload) (TelegramPayload, erro
 
 func createTelegramPayload(message string) TelegramPayload {
 	return TelegramPayload{
-		Message:           strings.TrimSpace(message),
+		Message:           markup.Sanitize(strings.TrimSpace(message)),
```

Comment:
Maybe `html.EscapeString()` for each fields (author, commit message) is better than santize of final message?
This santizer will not process the commit messages properly... Like that
`git commit -am 'Implemented some <a href="https://im.hacker">feature</a>'`

Reply:
That's the old behavior (I do not want to break too much in 1.22)

The complete fix is in  Refactor webhook #31587 , it does what you suggested.

Reply:
Ok, understood! 👍